hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java

```diff
@@ -166,6 +166,7 @@ public String convertToLetter(int size) {
         parameterCreator.addParam("cjv", Integer.toString(clusterClientStats.get(ClientType.JAVA)));
         parameterCreator.addParam("cnjs", Integer.toString(clusterClientStats.get(ClientType.NODEJS)));
         parameterCreator.addParam("cpy", Integer.toString(clusterClientStats.get(ClientType.PYTHON)));
+        parameterCreator.addParam("cgo", Integer.toString(clusterClientStats.get(ClientType.GO)));
```

Comment:
This should also be in master branch, right?

Reply:
Yes, I will forward-port these cahnges to master 👍

Reply:
@mustafaiman here is the forward port PR --> https://github.com/hazelcast/hazelcast/pull/12719